packages/salesforcedx-sobjects/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "lib": ["dom", "es6"],
+    "sourceMap": true,
+    "declaration": true,
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "rootDir": ".",
+    "outDir": "out",
+    "preserveConstEnums": true,
+    "strict": true
+  },
+  "exclude": ["node_modules", ".vscode-test", "out"]
```

Comment:
Since you are not a vscode module, you won't have a .vscode-test folder. So you can take that out.